airbyte-webapp/src/components/connection/CatalogTree/next/StreamPathSelect.tsx

```diff
@@ -61,14 +61,14 @@ export const StreamPathSelect: React.FC<PathPopoutProps> = (props) => {
     <PillSelect
       disabled={props.disabled}
       variant={props.variant}
-      className={styles.pillSelect}
       options={options}
       value={props.path}
       isMulti={props.isMulti}
       onChange={(options: PathPopoutProps["isMulti"] extends true ? Array<{ value: Path }> : { value: Path }) => {
         const finalValues = Array.isArray(options) ? options.map((op) => op.value) : options.value;
         props.onPathChange(finalValues);
       }}
+      className={styles.streamPathSelect}
```

Comment:
again, adjusting this to be full column-width.  not related to the main purpose of this PR, but a small, quick fix.